plugins/modules/python_distribution.py

```diff
@@ -126,6 +145,12 @@ def main():
             else:
                 desired_attributes["content_guard"] = None
 
+        if remote_name is not None:
+            if remote_name:
```

Comment:
```suggestion
        if remote_name:
```
Both empty string `""` and `None` are falsey values.

Reply:
remove the extra if